apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -340,6 +327,17 @@ class VotePanelContent extends React.Component {
   }
 }
 
+const ResponsiveVotePanelContent = props => (
+  <React.Fragment>
+    <BreakPoint to="medium">
+      <VotePanelContent {...props} shorten />
```

Comment:
Similar to https://github.com/aragon/aragon/pull/548/files#r249327353, but what do you think about making this more responsive as `shortAddresses`?